app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_create :increase_post_counter
```

Comment:
- [ ] Great job including the use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html). :clap: 
- [ ] However, in this instance, I will suggest you make use of  the `after_save` callback instead to call the `#increase_post_counter` method every time a new post has been saved to the database, then make the `increase_post_counter` method private :+1: 
- [ ] This is because the `after_create` callback is triggered after an object is created and saved to the database for the first time. It is only invoked when a new record is created and not when an existing record is updated. Therefore, it will not be triggered on subsequent updates to the same record :thinking: 
- However, the `after_save` callback, is triggered every time an object is saved to the database, regardless of whether it is a new record or an existing one being updated. It is invoked both after the creation of a new record and after updating an existing record and because of the nature of the blog app and the purpose of this `increase_post_counter` method, it will be more efficient to use the `after_save` callback instead :+1: 
- I will also suggest you use that in the other models as well :wink: